release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -2584,6 +2584,10 @@ To work around this issue, before you update to {product-title} {product-version
 
 * There is a disk discovery delay when attaching storage to workloads. (link:https://issues.redhat.com/browse/OCPBUGS-11149[*OCPBUGS-11149*])
 
+* When upgrading from {product-title} 4.12 to 4.13, Mellanox NIC renames SR-IOV network node policies such as `ens7f0` to `ens7f0np0`. Renaming the network node policies causes the `SriovNetworkNodePolicy` object to not be able to locate the original interface. Consequently, virtual functions (VFs) cannot be created because the interface cannot be found.
++
+To work around this issue, you must manually delete and re-add a SR-IOV network node policy with the new name after updating to {product-title} {product-version}. (link:https://issues.redhat.com/browse/OCPBUGS-11677[*OCPBUGS-13186*])
```

Comment:
```suggestion
To work around this issue, you must manually delete and re-add an SR-IOV network node policy with the new name after updating to {product-title} {product-version}. (link:https://issues.redhat.com/browse/OCPBUGS-11677[*OCPBUGS-13186*])
```

Reply:
Oh - just realized after I submitted. The link text doesn't match the URL (`OCPBUGS-11677[*OCPBUGS-13186*]`)